extensions/panache/hibernate-orm-panache/runtime/src/main/java/io/quarkus/hibernate/orm/panache/runtime/PanacheQueryImpl.java

```diff
@@ -24,25 +27,74 @@
     private static final Pattern SELECT_PATTERN = Pattern.compile("^\\s*SELECT\\s+((?:DISTINCT\\s+)?[^\\s]+)\\s+([^\\s]+.*)$",
             Pattern.CASE_INSENSITIVE);
 
-    private Query jpaQuery;
     private Object paramsArrayOrMap;
     private String query;
+    private String countQuery;
+    private String sort;
```

Comment:
this name feels wrong, isn't it an order-by clause? How about `orderByPart`?

Reply:
Yes simply `orderBy` feels better